src/utils/hooks/useFetchInstitutions.tsx

```diff
@@ -0,0 +1,43 @@
+import { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { InstitutionUnitBase } from '../../types/institution.types';
+import { getInstitutions } from '../../api/institutionApi';
+import { setNotification } from '../../redux/actions/notificationActions';
+import { NotificationVariant } from '../../types/notification.types';
+import Axios from 'axios';
+import { RootStore } from '../../redux/reducers/rootReducer';
+import { setInstitutions } from '../../redux/actions/institutionActions';
+
+// This hook should only be used when fetching the hierarchy of a given unit and not if it is desired
+// to access all subunits of the given unit.
```

Comment:
mener du at den skal brukes for å hente top level parent? er det hierarki nedover eller oppover? kan misforstås

Reply:
Hierarki oppover ja. kan legge til at det er bottom-up

Reply:
men responsen er top-down da, hvis du skjønner

Reply:
skjønner. men du kan godt skrive det sånn 👍